### ISSUE TITLE
False positive when multiplying against a negative number

### ISSUE THREAD

**Issue by reillywatson** [charithe/durationcheck#2]
```go
package dur

import "time"

const OneHourAgo = -1 * time.Hour
```
Outputs: "dur.go:5:20: Multiplication of durations: `-1 * time.Hour`"
